### DISCUSSION TITLE
Trigger IRSEND command with ALEXA

### DISCUSSION THREAD

**Discussion by sivabala21** [arendst/Tasmota#20462] [Q&A] [answered]
Can we trigger IRsend command with aleaxa routine or by saying Turn on AC to trigger Tasmota IR command ?
And I need to know how can we Store AC remote protocol and data  and trigger IRsend with stored data without home assistant !

**Comment by barbudor** [comment, marked as answer]
For the first part you can use a virtual relay, i.e. a GPIO connected to nothing and defined as a Relay.
Tasmota won't know that there is no hardware connected to it a'd it can be controlled by any method including Alexa, MQTT, Matter etc

Then you can have a couple of rules that triggers on power state changes:
```
Rule1
  On power1#state==0 do irsend... endon
  On power1#state==1 do irsend... endon
```
Insert above the proper irsend parameters for off and on

**Reply by sivabala21** [reply]
Let me try and update you !.
